docs/getting-started.md

```diff
@@ -115,96 +167,387 @@ After creation, your CVM will go through several states:
 
-#### Creating Assets
+Each role owner needs to create their asset (algorithm, dataset, etc.) and link it to the computation. Assets are cryptographically verified using file hashes and secured using your public/private key pair.
+
+> Note: While you create and link assets through the Prism web interface, you require the [Cocos CLI tool](https://github.com/ultravioletrs/cocos/releases) to perform certain operations like uploading files and retrieving results. As this connects you to the CVM and enclave agent via the enclave agent url. Make sure you have the Cocos CLI [installed](#prerequisites) before proceeding.
+
+---
+
+## Step-by-Step: Building Your Iris Classification Computation
+
+Now let's walk through creating and executing your computation.
+
+### Prepare Your Demo Files
+
+For this demo, you'll use:
+
+**Algorithm**: [Logistic Regression script](https://github.com/ultravioletrs/cocos/blob/main/test/manual/algo/lin_reg.py) `lin_reg.py`
+
+**Dataset**: [Iris flower measurements](https://github.com/ultravioletrs/cocos/blob/main/test/manual/data/iris.csv) (`iris.csv`)
+
+Save these files locally.
+
+### Generate File Hashes
+
+For each file, generate its cryptographic hash using the [Cocos CLI](https://docs.cocos.ultraviolet.rs/cli#command-checksum) to be used in the next step:
+
+```bash
+# Hash the algorithm
+./build/cocos-cli checksum lin_reg.py
+
+# Hash the dataset
+./build/cocos-cli checksum iris.csv
+```
+
+![Generate Hashes](../static/img/generate_hashes.png)
+
+> **Why hashes?** The hash is a unique digital fingerprint that ensures the exact file you specify is used in the computation and hasn't been modified.
+
+### Create Algorithm Asset
+
+1. Navigate to **Assets** → **Create New Asset**
+2. Select **Algorithm** type
+3. Fill in:
+
+   - **Name**: "Iris Classification Model"
+   - **Description**: "Logistic Regression for iris species classification"
+   - **File Hash**: Paste the hash from `lin_reg.py`
+
+      ![New Asset](../static/img/ui/new_asset.png)
+
+4. Click **Create Asset**
+
+### Link Algorithm to Computation
+
+1. From your **Assets** page, find "Iris Classification Model"
+
+   ![Algo Assets](../static/img/ui/algo_assets.png)
 
-Users assigned to roles must create their respective assets:
+2. Click **Associate**
 
-1. Navigate to **Assets** section
-2. Click **Create New Asset**
-3. Choose asset type (Algorithm, Dataset, etc.)
-4. Upload an optional sample of the asset
+   ![Associate Algo](../static/img/ui/algo_associate.png)
 
-![New Asset](../static/img/ui/new_asset.png)
-![User Assets](../static/img/ui/user_assets.png)
+3. Select your computation (e.g., "Iris Classification Demo")
+4. Confirm
 
-#### Linking Assets to Computations
+   ![Associate Asset](../static/img/ui/associate_algo_asset.png)
 
-1. From your assets page, find the relevant asset
-2. Click **Link to Computation**
-3. Search and select the target computation
-4. Confirm the association
+### Create Dataset Asset
```

Comment:
The dataset used in this section is incorrect: iris.csv cannot be used with the addition demo. This demo should use the iris algorithm together with the iris dataset, as the current ‘simple addition’ description no longer applies.